Models/ValidateRomResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MMRando.Models
+{
+    public enum ValidateRomResult
+    {
+
+        NoFile,
+        ValidFile,
+        InvalidFile,
+        Swap32,
+        Swap16
```

Comment:
Semantically, swap32 and swap16 do not describe whether the Rom is (in)valid, but that it requires to be (or is) swapped?

Reply:
It describes the "format" of the file.